tests/Unit/BankDataConverterTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace WMDE\Fundraising\Tests\Unit;
+
+use WMDE\Fundraising\Frontend\BankDataConverter;
+use WMDE\Fundraising\Frontend\Domain\BankData;
+use WMDE\Fundraising\Frontend\FunFunFactory;
+use WMDE\Fundraising\Frontend\Tests\TestEnvironment;
+
+/**
+ * @covers WMDE\Fundraising\Frontend\BankDataConverter
+ *
+ * @licence GNU GPL v2+
+ * @author Christoph Fischer <christoph.fischer@wikimedia.de
+ */
+class BankDataConverterTest extends \PHPUnit_Framework_TestCase {
+
+	/** @var FunFunFactory */
+	private $factory;
+
+	public function setUp() {
+		if ( !function_exists( 'lut_init' ) ) {
+			$this->markTestSkipped( 'The konto_check needs to be installed!' );
+		}
+		$this->factory = TestEnvironment::newInstance()->getFactory();
+	}
+
+	public function testWhenUsingConfigLutPath_constructorCreatesConverter() {
+		$this->assertInstanceOf( 'WMDE\Fundraising\Frontend\BankDataConverter', $this->factory->newBankDataConverter() );
```

Comment:
As of PHP 5.5 you can avoid using string class names. http://php.net/manual/de/migration55.new-features.php#migration55.new-features.class-name

So this could be `BankDataConverter::class`.